cop-import-export.php

```diff
@@ -0,0 +1,21 @@
+
```

Comment:
This should be in the same file as always have been.
It's hard to analyse what's different in a different file.

PS: A `<form>` inside a `<p>`? Is that valid?